site/ko/blog/new-in-devtools-105/index.md

```diff
@@ -101,31 +107,39 @@ Chromium issues: [1300613](https://crbug.com/1300613)
 
 
 <!-- ## Top layer badge in the Elements panel {: #top-layer } -->
-
+## 요소 패널의 Top layer 배지 {: #top-layer }
 <!-- Use the [top layer badge](/blog/top-layer-devtools/#top-layer-support-design-in-devtools) to understand the concept of the top layer and visualize how the top layer content changes.  -->
+[top layer badge](/blog/top-layer-devtools/#top-layer-support-design-in-devtools) 를 사용하여, top layer의 개념을 이해할 수 있으며 또한 top layer 의 컨텐츠 변화를 시각화할 수 있습니다.
```

Comment:
Capitalizing `Top layer badge` is also nice.

Reply:
Applied